docs/garage.md

```diff
@@ -0,0 +1,350 @@
+BENTO_DROP_BOX_S3_BUCKET="drop-box"                     # Created by init-garage
+BENTO_DROP_BOX_S3_REGION_NAME="garage"                  # Must match garage.toml
+BENTO_DROP_BOX_S3_ACCESS_KEY="<from-init-garage>"       # Save from init-garage output
+BENTO_DROP_BOX_S3_SECRET_KEY="<from-init-garage>"       # Save from init-garage output
+BENTO_DROP_BOX_VALIDATE_SSL=false                       # Set to false for self-signed certs
+```
+
+Restart Drop Box:
+
+```bash
+./bentoctl.bash restart drop-box
+```
+
+### 7. Configure DRS Service
+
+Add the following DRS configuration to your `local.env` file:
+
+```bash
+# local.env
+
+# DRS S3 Configuration
+BENTO_DRS_S3_ENDPOINT="garage.bentov2.local"            # Access via gateway
+BENTO_DRS_S3_USE_HTTPS=true                             # HTTPS through gateway
+BENTO_DRS_S3_BUCKET="drs"                               # Created by init-garage
+BENTO_DRS_S3_REGION_NAME="garage"                       # Must match garage.toml
+BENTO_DRS_S3_ACCESS_KEY="<from-init-garage>"            # Save from init-garage output
+BENTO_DRS_S3_SECRET_KEY="<from-init-garage>"            # Save from init-garage output
+BENTO_DRS_VALIDATE_SSL=false                            # Set to false for self-signed certs
+```
+
+Restart DRS:
+
+```bash
+./bentoctl.bash restart drs
+```
+
+## Configuration Reference
+### Domain Configuration
+
+```bash
+# Domain for accessing Garage S3 API through the gateway
+BENTO_GARAGE_DOMAIN=garage.${BENTOV2_DOMAIN}
+```
+
+This domain is used for:
+
+- **S3 API access**: `https://garage.bentov2.local/bucket/object` (path-style)
+
+### Security Configuration
+
+**For Development** (from `etc/bento_dev.env`):
+
+```bash
+# IMPORTANT: RPC secret must be exactly 64 hexadecimal characters (32 bytes)
+# Generate with: openssl rand -hex 32
+BENTO_GARAGE_RPC_SECRET=41f2965d423fe5c3904a76f7c7ab4ba186072fb4d7faf28643ddc8aaa3129e13
+BENTO_GARAGE_ADMIN_TOKEN=devgarageadmin789
+```
+
+**For Production** (set in `local.env` or override in `etc/bento_deploy.env`):
+
+```bash
+# Generate with: openssl rand -hex 32
+BENTO_GARAGE_RPC_SECRET=<your-generated-64-char-hex-secret>
+BENTO_GARAGE_ADMIN_TOKEN=<your-secure-admin-token>
+```
+
+> ⚠️ **Important**: The RPC secret must be exactly 64 hexadecimal characters (32 bytes). Generate with `openssl rand -hex 32`.
+
+**Best Practices:**
+
+- **Development**: Use the valid default in `bento_dev.env`
+- **Production**: Generate new secrets and set in `local.env` (never commit to git)
+- Both secrets should be treated as sensitive credentials
+
+### Garage Configuration File
+
+The `garage.toml` file is automatically generated by `init-garage` from the template at `etc/templates/garage/template.toml`. It substitutes environment variables and stores the result at `lib/garage/config/garage.toml`:
+
+```toml
+metadata_dir = "/var/lib/garage/meta"
+data_dir = "/var/lib/garage/data"
+
+replication_mode = "1"  # Single node
+
```

Comment:
should be `replication_factor`. Also, `consistency_mode` is missing here.

Reply:
removed it due to duplication